### PR TITLE
[codex] add Ark receive and transfer status CLI

### PR DESCRIPTION
**Pull request by bhandras**
## Summary

This PR makes the Ark CLI receive/status story usable without forcing users to know which lower-level protocol path they are exercising or to inspect daemon internals.

Today the CLI can create a boarding address, but there is no friendly generic receive command and no single place to inspect in-round/OOR transfer activity. That makes quick local testing awkward: users can generate a receive script, but then have to infer what happened by stitching together other command output.

This adds:

- `darepo receive` as the friendly default receive target command.
- `darepo receive list` to show locally known reusable receive/boarding targets.
- `darepo transfers list` to show in-round and OOR sends/receives with status.
- filters for transfer mode, direction, status, limit, and offset.
- daemon RPCs backing those CLI surfaces: `ListReceiveScripts` and `ListTransfers`.
- a bech32m address in `NewReceiveScriptResponse`, so callers no longer need to derive it from the returned script.

## User-Facing Examples

Create a generic Ark receive target:

```bash
darepocli receive
```

The command uses the same receive-script machinery as OOR receive target creation, but exposes it under the simpler command name users expect when they just want an address to hand out.

Create the same kind of receive target through the existing OOR namespace:

```bash
darepocli oor receive
```

Both commands return the pkScript plus the derived bech32m address. Existing scripts that call `oor receive` keep working, while new local-testing flows can use `receive` directly.

List receive targets this wallet/daemon knows about:

```bash
darepocli receive list
```

This is useful after a test session where multiple boarding/receive addresses were generated and you want to recover the exact script/address pairs without digging through logs.

List all transfer activity:

```bash
darepocli transfers list
```

List only pending in-round activity:

```bash
darepocli transfers list --mode inround --status pending
```

List only completed OOR receives:

```bash
darepocli transfers list --mode oor --direction incoming --status completed
```

Page through a larger history:

```bash
darepocli transfers list --limit 25 --offset 25
```

The transfer rows include stable identifiers, mode, direction, status, amount, timestamps where available, and context such as txid/round ID/error text depending on the source record.

## API Changes

- `NewReceiveScriptResponse` now includes `address`, derived from the taproot pkScript and the active network parameters.
- `ListReceiveScripts` returns the local receive targets indexed by the daemon.
- `ListTransfers` returns a normalized view over:
  - pending non-terminal in-round operations from round status tracking,
  - completed/failed in-round rows from transaction history,
  - OOR send/receive sessions from OOR session tracking.

The transfer list intentionally normalizes these sources into one response so CLI callers do not need separate commands for “in-round send”, “in-round receive”, “OOR send”, and “OOR receive” just to answer “what happened?”

## Current Limitations

`ListTransfers` aggregates several existing status/history sources in memory and then applies the public `limit`/`offset` page. That is expected to be fine for the small local-wallet histories this CLI is aimed at today, but it is not yet a database-level merged cursor for very large archival wallets. The code carries a TODO to replace the full scan with source-aware cursors if transfer history grows enough to need that.

Some rows can only expose the data available from their source. Live in-round rows may have unknown direction until local input/output hints or persisted ledger rows exist. Direction filters keep those unknown live rows visible rather than hiding pending activity behind `--direction incoming` or `--direction outgoing`. OOR receive/session rows can also report `amount_sat = 0` when the local status source does not yet know the final amount. The CLI still shows these rows so users can track lifecycle status instead of losing pending activity entirely.

## Schema Registry

The method registry now includes schemas for:

- `receive`
- `receive.list`
- `transfers.list`

It also refreshes the stale `send.oor` schema so it advertises the actual destination inputs: exactly one of `to` or `pubkey`. The removed `pk_script` schema entry was not accepted by the command parser, so this is a registry correction rather than a daemon/API removal.

## Validation

- `go test ./...`
- `make fmt-changed`
- `make fmt-changed-check`
- `make lint-native`
- `make commitmsg-lint range="origin/main..HEAD"`
- `make schema-check`
- `make doc-check` (passes; reports pre-existing CLAUDE.md/AGENTS.md divergence warnings in `baselib/actor` and `db/actordelivery/migrations`)
- `git diff --check`

### CODE REVIEW

File: darepod/rpc_transfers.go [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+		)
+		if err != nil {
+			return nil, err
+		}
+
+		transfers = append(transfers, history...)
+	}
+
+	if transferModeAllowed(req, daemonrpc.TransferMode_TRANSFER_MODE_OOR) {
+		oorTransfers, err := r.oorTransfers(ctx)
```
[the remark this answers is not in the document]
Comment (bhandras) [reply darepod/rpc_transfers.go]: Fixed and folded into `1ef001da`. `historyTransfers` now walks `ListTransactions` source pages until exhaustion, and `pendingRoundTransfers` walks `ListRounds` cursor pages instead of truncating at the first page.

File: darepod/rpc_transfers.go [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+	if err != nil {
+		return nil, err
+	}
+
+	sortTransfersNewestFirst(transfers)
+
+	start := int(req.GetOffset())
+	if start > len(transfers) {
+		start = len(transfers)
+	}
```
[the remark this answers is not in the document]
Comment (bhandras) [reply darepod/rpc_transfers.go]: Fixed and folded into `1ef001da`. `ListTransfers` now converts the uint32 cursor through `clampedTransferStart`, which compares using uint64 before converting to `int`.

File: cmd/darepocli/darepoclicommands/cmd_receive.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+	defer conn.Close()
+
+	req := &daemonrpc.ListReceiveScriptsRequest{}
+	if err := parseRequest(cmd, req, func() error {
+		return nil
+	}); err != nil {
+		return err
+	}
+
+	resp, err := client.ListReceiveScripts(context.Background(), req)
```
[the remark this answers is not in the document]
Comment (bhandras) [reply cmd/darepocli/darepoclicommands/cmd_receive.go]: Fixed and folded into `1ef001da`. The new receive-list RPC call now uses `cmd.Context()` so CLI cancellation propagates.

File: cmd/darepocli/darepoclicommands/cmd_transfers.go [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+		offset, _ := cmd.Flags().GetUint32("offset")
+		req.Offset = offset
+
+		return nil
+	}); err != nil {
+		return err
+	}
+
+	resp, err := client.ListTransfers(context.Background(), req)
```
[the remark this answers is not in the document]
Comment (bhandras) [reply cmd/darepocli/darepoclicommands/cmd_transfers.go]: Fixed and folded into `1ef001da`. The transfer-list RPC call now uses `cmd.Context()` so CLI cancellation propagates.

File: darepod/rpc_transfers.go [hunk 5]
```diff
@@ -0,0 +1,434 @@
...
+	status daemonrpc.TransferStatus) *daemonrpc.TransferInfo {
+
+	mode := daemonrpc.TransferMode_TRANSFER_MODE_INROUND
+	direction := daemonrpc.
+		TransferDirection_TRANSFER_DIRECTION_UNSPECIFIED
+
+	id := round.GetRoundId()
+	if id == "" {
+		id = fmt.Sprintf("temp:%t:%s", round.GetIsTemp(),
+			round.GetState().String())
```
[the remark this answers is not in the document]
Comment (bhandras) [reply darepod/rpc_transfers.go]: Fixed and folded into `1ef001da`. Temporary round rows now use `temporaryRoundTransferID`, which includes round state, timestamps, tx/outpoint context, and a per-list index to avoid collisions before a server round id exists.

**Comment by bhandras** [comment]
Addressed the first Claude review in `e17d5f03`:

- added the missing `requireWalletReady()` guard to `ListTransfers`
- changed transfer source collection to walk paginated `ListRounds`/`ListTransactions` sources instead of truncating at the first page
- inferred pending round direction from persisted local input/output outpoints when available, with an explicit comment for live rounds where direction is not yet knowable
- split the generic `receive` command through its own wrapper while sharing the NewReceiveScript helper with `oor receive`
- cleaned up the formatting/test/context nits and kept the fixes folded into the single branch commit

Re-running Claude via the existing `claude-review` label/synchronize workflow now.

**Comment by bhandras** [comment]
Folded the second Claude pass into `5b45e3b3` as well:

- documented `TransferInfo.direction` and CLI direction filtering for live in-round rows whose direction is still unknown
- added a code comment explaining the current in-memory aggregation tradeoff before the stores have a shared pagination model
- replaced the raw `"round"` history type with a named constant
- expanded parser tests for `all`/empty passthrough and added round-state status mapping coverage
- kept the `UpdatedAtUnixS = CreatedAtUnixS` assignment documented as intentional for immutable history rows

Local gate is green again after regenerating protobuf stubs.

**Comment by bhandras** [comment]
Folded the remaining third-pass clarity nits into `d600f196`:

- documented the possible round-status/history snapshot race while keeping source-prefixed transfer ids distinguishable
- changed transaction direction inference to use ledger event/account constants instead of substring matching
- removed the no-op `receive list` parse closure
- made the taproot key slice bound explicit
- added sort tie-break coverage for `sortTransfersNewestFirst`

Local full gate is green again: `go test ./...`, `make lint-native`, `make fmt-changed-check`, `make commitmsg-lint range="origin/main..HEAD"`, `make schema-check`, `make doc-check`, and `git diff --check`.

**Comment by bhandras** [comment]
Folded the fourth Claude pass into `5d6926ef`:

- refactored pending round collection into explicit live-state and persisted-round traversals so the `PersistedOnly` cursor behavior is no longer subtle
- documented why terminal persisted round summaries are dropped in favor of transaction history rows
- added the `TODO` on the current O(total history) in-memory aggregation path
- added a `Long` description for `receive list`
- aligned the CLI transfer limit default with the RPC default (`100`)
- documented zero `amount_sat` for live in-round rows until history supplies the persisted amount

Full local gate is green again, including regenerated protobuf stubs.

**Comment by bhandras** [comment]
Folded the fifth Claude pass into `996974a3`:

- added round-id deduplication for in-round rows, preferring terminal/history rows over pending live rows when both describe the same persisted round
- added unit coverage for that dedupe behavior
- documented OOR session rows as another case where `amount_sat` may remain zero until history supplies a persisted amount
- changed uninitialized indexer handling in `ListReceiveScripts` from `Internal` to `FailedPrecondition`
- kept the receive-script list TODO and formatter-stable target allocation

Full local gate is green again after regenerating stubs.

**Comment by bhandras** [comment]
Folded the latest review pass into 9d890476c7263323fd1d24502c34f8f0911eb391. This adds transaction-history status mapping instead of hard-coding completed, covers failed status/subtype mapping in tests, parallelizes the table subtests, documents the OP_DATA_32 taproot-script offset, and explains the dedupe tie-break. I also expanded the PR body with concrete CLI examples, the full-scan limitation, and the rows that can currently report unknown direction or zero amount.\n\nLocal validation on the amended head: `make commitmsg-lint range="origin/main..HEAD"`, `go test ./...`, `make lint-native`, `make fmt-changed-check`, `make schema-check`, `make doc-check`, and `git diff --check`. `doc-check` still prints the two known pre-existing CLAUDE/AGENTS divergence warnings but exits successfully.

**Comment by bhandras** [comment]
Folded the latest Claude round into c01ed49c0. Changes in this pass:\n\n- replaced fuzzy `strings.Contains(subtype, "failed")` status inference with exact matching on the persisted failed transaction-history status constant\n- added test coverage that a subtype containing the word "failed" does not get misclassified as failed\n- corrected the `collectTransfers` comment so it matches where filtering actually happens\n- mapped receive-script indexer read failures to `codes.Unavailable`\n- expanded `transferMatchesFilters` coverage for nil rows, OOR mode mismatch, status mismatch, and unknown-direction live rows\n- changed direction filtering so unknown live in-round rows remain visible under an incoming/outgoing filter, with CLI help and PR-body text documenting that behavior\n- removed the untracked receive-script pagination TODO wording and cleaned up the transfer ID formatting\n\nLocal validation on this head: `make commitmsg-lint range="origin/main..HEAD"`, explicit commit-message shape check, `go test ./...`, `make lint-native`, `make fmt-changed-check`, `make schema-check`, `make doc-check`, and `git diff --check`. `doc-check` still reports the two pre-existing CLAUDE/AGENTS divergence warnings but exits successfully.

**Comment by bhandras** [comment]
Folded the latest Claude round into 2727a758. This pass adds the direction-filter contract directly to the proto docs and regenerates the daemon RPC stubs, so the `TRANSFER_DIRECTION_UNSPECIFIED` pass-through behavior is part of the API documentation instead of just CLI help text. It also documents the transient live-FSM/persisted-pending duplicate window in `pendingRoundTransfers`, including why rows with the same round id are deduped and why a temporary live row without a round id may briefly remain visible rather than hiding active local state.\n\nI left the defensive `TRANSFER_MODE_UNSPECIFIED` fallback in `transactionTransferID` but added an explanatory comment. Receive scripts remain indexer-backed here, so the initialized-indexer precondition stays explicit and indexer read failures return `codes.Unavailable`.\n\nLocal validation on this head: `make rpc`, `make commitmsg-lint range="origin/main..HEAD"`, explicit commit-message shape check, `go test ./...`, `make lint-native`, `make fmt-changed-check`, `make schema-check`, `make doc-check`, and `git diff --check`. `doc-check` still reports the two pre-existing CLAUDE/AGENTS divergence warnings but exits successfully.

**Comment by bhandras** [comment]
Folded the latest Claude pass into `65301939`. This round adds the `transfers list` long help text for the unknown-direction pass-through behavior, removes the unnecessary nil request guard in `ListTransfers`, narrows the history helper from a generic `historyType` parameter to explicit round history, and documents the input-priority direction heuristic plus the response-local nature of temporary live-round ids.

I also checked the module target (`go 1.25.5`) and removed the now-unneeded `test := test` loop captures in the touched table tests.

Local validation on this head: `go test ./darepod ./cmd/darepocli/darepoclicommands ./daemonrpc`, `make lint-native`, `make commitmsg-lint range="origin/main..HEAD"`, explicit commit-message shape check, `make fmt-changed-check`, and `git diff --check`.

**Comment by bhandras** [comment]
Folded the latest Claude review items into `d30e66ad`.

This round makes the `ListTransfersRequest.direction_filter` contract explicit that `TRANSFER_DIRECTION_UNSPECIFIED` rows are always included, documents the silent `limit` cap in the proto, and adds TODOs at the two known follow-up points: persisted pending-round scans should eventually get status-filter pushdown, and OOR amount display needs an amount source in `OORSessionInfo` or a history-backed merge.

I also renamed the temporary live-round id label from `temp:true` to `is_temp:true`, added coverage for `clampedTransferStart(offset == transferCount)`, and extended the round dedupe test to cover two completed history rows tying on status/time.

Local validation on this head: `make rpc`, `make fmt-changed`, `go test ./darepod ./cmd/darepocli/darepoclicommands ./daemonrpc`, `go test ./...`, `make lint-native`, `make commitmsg-lint range="origin/main..HEAD"`, explicit 3-10 sentence commit-message check, `make fmt-changed-check`, `make schema-check`, `make doc-check`, and `git diff --check`. `make doc-check` exited 0 with the pre-existing AGENTS/CLAUDE divergence warnings in `baselib/actor` and `db/actordelivery/migrations`.

**Comment by bhandras** [comment]
Folded the latest Claude review items into `c8e359f4`.

This round adds the missing `t.Parallel()` calls in `rpc_receive_targets_test.go`, switches taproot receive-address derivation to btcd's standard `txscript.IsPayToTaproot` / `ParsePkScript` / `Address` path instead of hand-checking opcodes and slicing the x-only key, sharpens the persisted-round scan TODO to call out the needed future `ListRoundsRequest.state_filter`, clarifies why the internal `TRANSFER_MODE_UNSPECIFIED` transaction id fallback emits a `ledger:` id without logging, and documents that new OOR statuses surface as `TRANSFER_STATUS_UNSPECIFIED` until the projection is updated.

Local validation on this head: `make fmt-changed`, `go test ./darepod ./cmd/darepocli/darepoclicommands ./daemonrpc`, `go test ./...`, `make lint-native`, `make commitmsg-lint range="origin/main..HEAD"`, explicit 3-10 sentence commit-message check, `make fmt-changed-check`, `make schema-check`, `make doc-check`, and `git diff --check`. `make doc-check` still exits 0 with the pre-existing AGENTS/CLAUDE divergence warnings in `baselib/actor` and `db/actordelivery/migrations`.

**Comment by bhandras** [comment]
Folded the latest hardening suggestions into `e9a6bed0`.

This pass keeps temporary live-round ids bounded by hashing the variable round fields and including only `is_temp`, the short digest, and the response-local index in the returned id. The test now asserts the temporary id remains short while still distinguishing two live rows from the same round snapshot.

I also expanded the persisted pending-round scan comment to cover the live-plus-persisted pending duplicate case, sharpened the `ListRoundsRequest.state_filter` TODO as a proto-level prerequisite, and documented that transaction direction inference is coupled to the ledger package's canonical event/account labels.

Local validation on this head: `make fmt-changed`, `go test ./darepod ./cmd/darepocli/darepoclicommands ./daemonrpc`, `go test ./...`, `make lint-native`, `make commitmsg-lint range="origin/main..HEAD"`, explicit 3-10 sentence commit-message check, `make fmt-changed-check`, `make schema-check`, `make doc-check`, and `git diff --check`. `make doc-check` still exits 0 with the pre-existing AGENTS/CLAUDE divergence warnings in `baselib/actor` and `db/actordelivery/migrations`.

**Comment by bhandras** [comment]
Folded the latest Claude pass into `4a1ae46d1363cbb66925452ef00e36fb265ae555`.

This round makes the `transfers list` long help text more explicit that direction filters still return live in-round rows whose direction is unknown, adds a note that future history-backed OOR transfer rows should dedupe by session id, documents that transfer-list limits are applied after collection/filtering/deduplication to preserve global newest-first ordering, and adds coverage for the nil registered receive-script guard.

Local validation on this head: `make commitmsg-lint range="origin/main..HEAD"`, explicit 3-10 sentence commit-message check, `make fmt-changed-check`, `git diff --check`, `go test ./darepod ./cmd/darepocli/darepoclicommands ./daemonrpc`, `go test ./...`, `make lint-native`, `make schema-check`, and `make doc-check`. `make doc-check` still exits 0 with the pre-existing AGENTS/CLAUDE divergence warnings in `baselib/actor` and `db/actordelivery/migrations`.

**Comment by bhandras** [comment]
Folded the latest Claude findings into `ea11364cd6e9cb7c95b744b6dfb7ebec8093d4be`.

This pass makes the `receive --label` flag forwarding explicit and covered by `TestNewReceiveScriptRequestFromCmdForwardsLabel`, so the normal cobra path cannot silently drop the label. It also fixes round-history duplicate supersession to compare the numeric ledger entry suffix instead of lexicographic transfer-id strings, with the existing dedupe test updated to cover `round:abc:10` beating an older single-digit row.

I also documented that the failed transaction-history status string mirrors `TransactionHistoryEntry.confirmation_status`.

Local validation on this head: `make fmt-changed`, `go test ./darepod ./cmd/darepocli/darepoclicommands ./daemonrpc`, `go test ./...`, `make lint-native`, `make commitmsg-lint range="origin/main..HEAD"`, explicit 3-10 sentence commit-message check, `make fmt-changed-check`, `make schema-check`, `make doc-check`, and `git diff --check`. `make doc-check` still exits 0 with the pre-existing AGENTS/CLAUDE divergence warnings in `baselib/actor` and `db/actordelivery/migrations`.

**Comment by bhandras** [comment]
Folded the latest Claude cleanup pass into `cef870ab29298e0ed1e57b549227ed6d10aa6096`.

This pass removes the in-place transfer filter aliasing by building a fresh filtered slice, narrows `transferHistoryEntryID` into `roundHistoryEntryID` and adds table coverage for round/OOR/ledger/malformed ids, aligns the new CLI connection cleanup with the explicit close-discard pattern used in the package, switches the source offset cap to `math.MaxInt32`, and documents that the exhaustive persisted-round scan keeps `ListTransfers.HasMore` accurate until the source query can filter by round state.

I also added a success-path test for `receiveTargetFromRegisteredScript` and regenerated the daemon protobuf stubs for the `ListTransfersResponse` comment that warns response rows can still include unknown-direction entries under a direction filter.

Local validation on this head: `make rpc`, `make fmt-changed`, `go test ./darepod ./cmd/darepocli/darepoclicommands ./daemonrpc`, `go test ./...`, `make lint-native`, `make commitmsg-lint range="origin/main..HEAD"`, explicit 3-10 sentence commit-message check, `make fmt-changed-check`, `make schema-check`, `make doc-check`, and `git diff --check`. `make doc-check` still exits 0 with the pre-existing AGENTS/CLAUDE divergence warnings in `baselib/actor` and `db/actordelivery/migrations`.

**Comment by bhandras** [comment]
Folded the latest Claude review into `93885070af52e9e25992af8d2e23b953c9827bcb`.

This pass addresses the new review items directly: `collectTransfers` now skips the persisted pending-round scan whenever the request explicitly filters for terminal statuses, `ListReceiveScripts` has an explicit pagination TODO for future larger receive-target registries, and the transfer response proto comment now warns downstream callers not to rely on unknown-direction pass-through as permanent exclusive-filter semantics.

I also added targeted tests for the transfer dedupe/supersession path and expanded the newest-first sort test so it covers the zero-`updated_at_unix_s` fallback against populated update times.

Local validation on this head: `make rpc`, `make fmt-changed`, `go test ./darepod ./daemonrpc`, `go test ./...`, `make lint-native`, `make commitmsg-lint range="origin/main..HEAD"`, explicit 3-10 sentence commit-message check, `make fmt-changed-check`, and `git diff --check`.

Claude, please re-review this latest head.

**Comment by bhandras** [comment]
Folded one more Claude follow-up pass into `2457386b4b020d76afa94457fe3b8c6505016c80`.

This pass adds handler-level readiness tests for both new RPCs, expands the transfer direction test with the subtype-vs-account priority case, verifies the schema registry keeps `limit` and `offset`, tightens the `--direction` flag help so unknown pending in-round rows are called out directly, and makes the `transactionTransferID` fallback arm explicit with `default`.

I left the full-history scan as the documented local-wallet tradeoff for this PR; the code already skips the pending-round scan for terminal status filters and the remaining O(total history) behavior is called out in the implementation comments for a future source-level pagination/filter pushdown.

Local validation on this head: `make fmt-changed`, `go test ./darepod ./cmd/darepocli/darepoclicommands`, `go test ./...`, `make lint-native`, `make commitmsg-lint range="origin/main..HEAD"`, explicit 3-10 sentence commit-message check, `make fmt-changed-check`, and `git diff --check`.

Claude, please re-review this latest head.

**Comment by bhandras** [comment]
Folded the final Claude polish pass into `ea997fd6f4682f8b00ac5d052c577df9bcf18f09`.

This pass expands the transfer schema descriptions so tool callers can see the valid enum values directly, updates the transfer-list `--limit` help to mention its default behavior, keeps the schema descriptions covered by `TestMethodRegistryTransfersSchema`, and gives the source-pagination TODO an explicit tracking reference.

Local validation on this head: `make fmt-changed`, `go test ./cmd/darepocli/darepoclicommands ./darepod`, `go test ./...`, `make lint-native`, `make commitmsg-lint range="origin/main..HEAD"`, explicit 3-10 sentence commit-message check, `make fmt-changed-check`, and `git diff --check`.

Claude, please re-review this latest head.

**Comment by bhandras** [comment]
Folded the latest Claude review into `5e6c0be1c4a4de37a7e6e1aa0a3bff2ae223b539`.

This pass skips committed round-history scans for pending-only transfer requests, replaces the raw `"round"` transaction-history filter with the typed `ledger.TransactionTypeRound` constant, updates `transferFromRound` and transfer dedupe naming/comments, and moves the future OOR session-id dedupe TODO next to the dedupe logic. It also adds parser coverage for `--status pending` and `--status failed`, adds focused coverage for the history-scan status gate, and removes the now-unneeded loop-variable capture in the round history ID test.

Local validation on this head: `make fmt-changed`, `go test ./cmd/darepocli/darepoclicommands ./darepod ./ledger` with a clean rerun of `go test ./darepod` after a timing-only listener-test timeout, `go test ./...`, `GOMAXPROCS=2 GOGC=50 make lint-native`, `make commitmsg-lint range="origin/main..HEAD"`, explicit 3-10 sentence commit-message check, `make fmt-changed-check`, and `git diff --check`.

@claude, please re-review this latest head.

**Comment by bhandras** [comment]
@claude re-review

**Comment by bhandras** [comment]
Closing this stale prototype.

The branch explored an Ark receive and transfer status CLI, but it is not part of the current CLI roadmap and has drifted substantially from `main`. We do not plan to restack or continue this implementation.

If a concrete status-surface requirement returns, it should start from the current activity and wallet APIs as a focused change.
